include/swift/Option/FrontendOptions.td

```diff
@@ -883,6 +883,12 @@ def solver_scope_threshold_EQ : Joined<["-"], "solver-scope-threshold=">,
 def solver_trail_threshold_EQ : Joined<["-"], "solver-trail-threshold=">,
   HelpText<"Expression type checking trail change limit">;
 
+def solver_disable_prepared_overloads : Flag<["-"], "solver-disable-prepared-overloads">,
+  HelpText<"Disable experimental prepared overloads optimization">;
+
+def solver_enable_prepared_overloads : Flag<["-"], "solver-enable-prepared-overloads">,
+  HelpText<"Enable experimental prepared overloads optimization">;
+
```

Comment:
Why not make this an experimental feature so you don't have to extend LangOpts and add new flags?

Reply:
I could, but it’s not a language feature and it’s only temporary (hopefully) so it felt like a misuse of experimental features… hmm.